### DISCUSSION TITLE
Babylonian Calendar?

### DISCUSSION THREAD

**Discussion by Balderick** [Stellarium/stellarium#1666] [Ideas]
It really great that we can select Babylonian MUL.APIN constellations but would be even better if we had a Babylonian calendar too.

Can see the new Calendar plug in does have Hebrew calender but it seems to not be very accurate for older sky charts in ancient times.  

Have found this handy converter which does awesome job of converting Babylonian calendar dates to Julian and am finding it useful for doing what i am using Stellarium for.

Any plans for adding Babylonian calender to Calendar plug in so there is native way to show accurate Babylonian dates matching Julian equivalent inside Stellarium?

What would be involved for building a plug in to add Babylonian calendar ?

**Comment by gzotti** [comment]
The Calendars plugin is still under development. It is based on the calendars described algorithmically in the book mentioned in its "About" info. 
You have found "this handy converter". I have not. I do not know if there is an algorithmically describable calendar possible when it was (AFAIK) observation based. For me it has the same problem as the observation based Muslim calendar. There may be some algorithmic approximation available. If you can dig up such information, I can include it.

**Comment by bean32** [comment]
I have ordered the paperback version of the book Calendrical Calculations referenced in Stellarium. I'll begin looking into the issue and see if I can dig up some information on how to algorithmically account for cultural calendars based on lunar observations.   Some of the core issue to take into account is the change in methodology in ancient methods compared to modern ways of counting time based on the lunar calendar.

**Comment by jaaptitulaer** [comment]
Please be aware that there is no fixed Babylonian calendar prior to a certain date, and AFAIK certainly not before ca 747 BC.
In fact before some time in the 7th century or so more likely (650? 680?). Before that time the lunar intercalation was ad-hoc and not some fixed approach. I think there is a book which should cover the reconstructed correction (i.e. when the intercalations were done) up to some earlier time, maybe mid 8th century, I can't remember exactly.
Before that and certainly before the 1st millennium you will have to do with the knowledge that the intercalation is uncertain and needs to be determined by ancient document matched to astronomical observations. And in that way working back into time. But you'll quickly cross 'dark age' periods of contested history and little or even no documentation, which complicates matters immensely.

**Reply by gzotti** [reply]
That's the point. Some authors may have found what they believe are best probable solutions. Are they really "true"? It is probably beyond my competence to fix Babylonian chronology, a topic with at least decades of high-effort research.

**Reply by jaaptitulaer** [reply]
Doing just that :)

I rechecked that book/paper, their main work was: https://www.staff.science.uu.nl/~gent0113/babylon/downloads/babylonian_chronology_pd_1956.pdf
https://oi.uchicago.edu/research/publications/saoc/saoc-24-babylonian-chronology-626-bc-%E2%80%93-ad-45

Babylonian Chronology, 626 B.C. – A.D. 45
Richard A. Parker and Waldo H. Dubberstein

This study aims at providing a brief, but complete and thorough, presentation of the data bearing upon the chronological problems of the Neo-Babylonian, Achaemenid Persian, and Seleucid periods, together with tables for the easy translation of dates from the Babylonian calendar into the Julian. Recent additions to our knowledge of intercalary months in the Neo-Babylonian and Persian periods have enabled us to improve upon the results of our predecessors in this field, though our great debt to F. X. Kugler and D. Sidersky for providing the background of our work is obvious. While our tables are intended primarily for historians, both classical and oriental, biblical students also should find them useful, as any biblical date of this period given in the Babylonian calendar can be translated by our tables. [From SAOC 24, "Preface," p. vii, by Parker and Dubberstein]
----

I thought they later went beyond that, but I'm not sure. Upto 747 BC could perhaps be done, because that's mostly a known chronology period with lots of documentation. First period of uncertainty starts in 627 BC (hence their original limit I think).
But say one can fix that then back to at best 800 BC when Ninurta-apla-X starts to reign. Before that an interregnum with 5 unknown kings. By then we're probably already in 'circa' dating and poor documentation.
Chronologists relink via eclipses and synchronisms, but that won't help you recreate a continuous Babylonian calendar, I'm afraid.

**Reply by jaaptitulaer** [reply]
Woops sorry for the big font via copy & paste.

**Reply by jaaptitulaer** [reply]
The reference implementation can be found at UU.nl
https://webspace.science.uu.nl/~gent0113/babylon/babycal.htm
With even a handy link to precomputed table:
https://webspace.science.uu.nl/~gent0113/babylon/downloads/babylonian_chronology_pd_1971.dat

**Reply by gzotti** [reply]
Thanks for the books and lists. It's not on my shortlist, though, but who knows? Or yet again, anybody else deeper connected into Babylonian details could do it...
